aries-site/src/data/structures/components.js

```diff
@@ -679,6 +679,9 @@ export const components = [
   },
   {
     name: 'CheckBox',
+    accessibility: 'Failed WCAG 2.2 A',
+    accessibility: 'Failed WCAG 2.2 AA',
+    accessibility: 'Failed WCAG 2.2 AAA',
```

Comment:
```suggestion
    accessibility: 'Failed WCAG 2.2 A',
```
We just need to say that it failed level A, don't need the other two messages